app/views/layouts/sumofus.slim

```diff
@@ -14,5 +14,7 @@ html
     = yield
 
     .mobile-indicator
-    - if current_user && @page.present?
-      = link_to 'Edit Page', edit_page_path(@page), class: 'campaigner-overlay__edit-link'
+    - if @page.present?
+      span.hidden.page_id = @page.id
```

Comment:
I'd like to associate the `page` object with any transaction/subscription:

https://github.com/SumOfUs/Champaign/pull/274 (wip)

I think if you just take the `Payment::BraintreeTransaction` active record instance, you'll have most of the information you need for posting to `SQS` (including the `page_id`).